CMakeLists.txt

```diff
@@ -78,8 +78,8 @@ if (${FAASM_CUSTOM_LLVM})
     set(WAVM_PERF_LIBS 1)
     add_definitions(-DWAVM_PERF_EVENTS=1)
 else ()
-    # Use LLVM 8 explicitly for the lib parts (i.e. codegen)
-    set(LLVM_DIR /usr/lib/llvm-8)
+    # Use LLVM 9 explicitly for the lib parts (i.e. codegen)
+    set(LLVM_DIR /usr/lib/llvm-9)
```

Comment:
Can we not just go straight to 10? https://github.com/WAVM/WAVM/blob/master/CMakeLists.txt#L108

Reply:
That's a typo, this line was deleted as per the first bullet point in the PR description I don't know what happened to that commit!

Reply:
Fixed in https://github.com/lsds/Faasm/pull/229/commits/0db948cb63fa1221aa13cb23f1c042024a6261bd

Reply:
So can we remove the llvm 9 playbook too and have cpp-root use 10 as well?

Reply:
I'm not 100% sure `libWAVM` works with `llvm-10` despite the comment. I thought it worked but when building llvm for perf I ran into issues with it not linking properly, it might just be a bug because my system was giving contradicting information but I wanted to play it safe in the docker build to not have you grinding your teeth when a CI later fails because `libWAVM` didn't link properly. Happy to change though if you think llvm-10 is safe.

Reply:
Nah, that's fair enough, very wise to play it safe. I had a look through the WAVM commit history and saw no evidence of any LLVM 10-related tweaks, so they've probably not got round to it yet. Should be good to go now.

Reply:
I rebuilt the containers with LLVM-10 and it works, I also tried to rebuild llvm perf with LLVM-10 and it's indeed still that one that doesn't work. As such I will not update to 10 in my changes to the llvm_perf script in #224 but have updated `cpp-root`